### ISSUE TITLE
Detecting unused accessors (getters & setters)

### ISSUE THREAD

**Issue by domsleee** [webpro/knip#297]
Hey, thanks for this great program 😄 

On version 2.33.3, I have found that it doesn't seem to detect unused getters and setters of a class

```ts
export class MyClass {
  public usedMethod() {
    console.log("I am used");
  }

  public unusedMethod(): string {
    return "unusedMethod";
  }

  public get unusedGetter(): string {
    return "unusedGetter";
  }

  public set unusedSetter(_value: string) {
    console.log('unusedSetter');
  }
}

```

```ts
const c = new MyClass();
c.usedMethod();
```

It outputs this:
```shell
Unused exported class members (1)
unusedMethod  MyClass  src/class.ts
```

I would have expected `unusedSetter` and `unusedGetter` to appear in the list.

Is this something `knip` could support?

P.S. It looks like #274 is about unused methods of objects, this one is specifically about getters/setters of classes. They are similar features 👍

**Comment by webpro** [comment]
This sounds like a missing feature, I'll look into it.

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.34.0. See [Release 2.34.0](https://github.com/webpro/knip/releases/tag/2.34.0) for release notes._
